mautic/mautic
# change link from edit to view once company modification saved

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✅ 
| New feature/enhancement? (use the a.x branch)      | 🔴 
| Deprecations?                          | 🔴 
| BC breaks? (use the c.x branch)        | 🔴 
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | not needed
| Related developer documentation PR URL | not needed
| Issue(s) addressed                     | not needed



#### Description:
Little UX improvement to have link in flash message when saving a company going to `view` page instead of `edit`. This bring the same behavior we see on contact, etc. Probably like this because in the past the view page was not existing for companies.

#### Steps to test this PR:


1. Create a company, save and close
2. Click the company name on the flash message --> you arrive on the edit page of the company
3. Apply PR, do the same and you should arrive on view page.

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11098"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
